pomegranate/hmm.pyx

```diff
@@ -2486,8 +2490,10 @@ cdef class HiddenMarkovModel(GraphModel):
                     initial_log_probability_sum = log_probability_sum
                 else:
                     improvement = log_probability_sum - last_log_probability_sum
+                    time_spent = time.time() - epoch_start_time
                     if verbose:
-                        print("Training improvement: {}".format(improvement))
+                        print("Training improvement: {} in {:.2f}s".format(improvement,
+                                                                           time_spent))
```

Comment:
Ultimately it doesn't matter too much. It is obvious what is going on to people who want to develop.